fdbserver/DataDistribution.actor.cpp

```diff
@@ -2459,8 +2534,77 @@ ACTOR Future<Void> teamRemover(DDTeamCollection* self) {
 	}
 }
 
+// Remove the server team whose members have the most number of process teams
+// until the total number of server teams is no larger than the desired number
+ACTOR Future<Void> serverTeamRemover(DDTeamCollection* self) {
+	state int numServerTeamRemoved = 0;
+	loop {
+		// In case the serverTeamRemover cause problems in production, we can disable it
+		if (SERVER_KNOBS->TR_FLAG_DISABLE_SERVER_TEAM_REMOVER) {
+			return Void(); // Directly return Void()
+		}
+
+		// To avoid removing machine teams too fast, which is unlikely happen though
+		double removeServerTeamDelay = SERVER_KNOBS->TR_REMOVE_SERVER_TEAM_DELAY;
+		if (g_network->isSimulated()) {
```

Comment:
try to change this in the knob file itself

Reply:
we cannot use g_network in knob file because it has not been initialized when knob is initialized. Using it will cause segmentation fault.